CHANGELOG.md

```diff
@@ -7,16 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Added support for new devices in the nRF52 family. nRF52805, nRF52820 and nRF52833.
+### Changed
+
+### Fixed
+
+## [0.8.0]
+
+### Added
+
+- Added support for new devices in the nRF52 family - nRF52805, nRF52820 and nRF52833.
 - Added support for the STM32F7 family.
+- The `Session` struct and dependants now implement `Debug`.
+- The J-Link driver now displays a warning if no proper target voltage is measured.
```

Comment:
```suggestion
- The J-Link driver now logs a warning if no proper target voltage is measured.
```

This is more accurate since it depends on the log level configuration (although warnings are shown by default).